test/endtoend/operator/101_initial_cluster_backup.yaml

```diff
@@ -15,12 +15,12 @@ spec:
           path: /backup
           type: Directory
   images:
-    vtctld: vitess/lite:v13.0.0
-    vtgate: vitess/lite:v13.0.0
-    vttablet: vitess/lite:v13.0.0
-    vtbackup: vitess/lite:v13.0.0
+    vtctld: vitess/lite:latest
```

Comment:
why is this latest and not v13.0.x?

Reply:
This file is used for the backup test and not the upgrade test. In the backup test we are testing that the backups work with the latest version of Vitess. Earlier I had 13.0.0 there, because the code on the operator was supposed to only work with 13.0.0. After this PR, the code will be meant to be working with the release-14 of Vitess, so we use the latest image.